libreflexxestype2/CMakeLists.txt

```diff
@@ -17,44 +13,47 @@ include_directories(
   ${catkin_INCLUDE_DIRS}
 )
 
-find_program(CMAKE_DPKG dpkg /bin /usr/bin /usr/local/bin)
-if(CMAKE_DPKG)
-  exec_program(dpkg ARGS --print-architecture OUTPUT_VARIABLE CMAKE_DPKG_ARCH)
-  if(CMAKE_DPKG_ARCH MATCHES "amd64")
-    message(STATUS "+++++++++++++++++ DETECTED 64 bit +++++++++++++++++")
-    set(SUPPORTED "true")
-  else()
-    message(WARNING "+++++++++++++++++ UNSUPPORTED ARCHITECTURE, UNABLE TO USE ReflexxesTypeII +++++++++++++++++")
-    hset(SUPPORTED "false")
-  endif()
-endif()
-
-if(SUPPORTED MATCHES "true")
-  # Set location of build output
-  set(libreflexxes_LOCATION ${CMAKE_CURRENT_SOURCE_DIR}/Linux/x64/release/lib/shared/libReflexxesTypeII.so)
-
-  # Build Reflexxes library with a Make file
-  add_custom_command(
-    OUTPUT ${libreflexxes_LOCATION}
-    COMMAND make -C "${CMAKE_CURRENT_SOURCE_DIR}/Linux" clean64 all64 WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
-  )
```

Comment:
Is there a makefile we should also delete?  this is really weird that a c++ library would be built with cmake and they add a custom command to call make.